### ISSUE TITLE
Graduation cap

### ISSUE THREAD

**Issue by designlanguage** [cfpb/design-manual#303]
I'm reposting this here:

While I was helping @Dnpizarro  prepare a storyboard, the issue of the graduation cap came up. Because of it's angle, it can be difficult to work with at times. It's a bit tilted forward, and since our illustration style is either front facing or top down, the cap is not always harmonious with other illustrations. Here's the current version:

![07451f10-d91f-11e4-85ca-7b3739578f54](https://cloud.githubusercontent.com/assets/10562544/6983744/973eaf66-d9ef-11e4-976b-9ba61fd0dd85.png)

Maybe we can make the cap more front-facing so it's easier to work with. Let me know if you think any of these explorations work.  Here's one option that's straight on:

![7b78ab72-d91f-11e4-9bdb-0c1ee66ebec9](https://cloud.githubusercontent.com/assets/10562544/6983745/9c9466ae-d9ef-11e4-84cc-4e00018f1305.png)

Here are two versions with a slight tilt, the first has a shadow on the rim of the cap:

![b80db5aa-d91f-11e4-8c41-db4e308140ad](https://cloud.githubusercontent.com/assets/10562544/6983751/a5fb0a5e-d9ef-11e4-99f2-c6c220d56f22.png)
![b3eb7142-d91f-11e4-95ac-8fe44657860c](https://cloud.githubusercontent.com/assets/10562544/6983759/b3281348-d9ef-11e4-8625-d4756d639405.png)

And here's the slightly tilted cap on its own:

![c9bbf83e-d91f-11e4-8082-e9b1d01ecba0](https://cloud.githubusercontent.com/assets/10562544/6983763/c17e524a-d9ef-11e4-8141-b49b710e3c39.png)

@caheberer suggested a half shadow on the brim and @Dnpizarro suggested a pointed front.

The pointed front can't go below the baseline or the same problem will exist of looking like it's tilted. So I tried this other approach, but it seems a little weird to me. The point feels too solid, almost like a helmet. The objective of these illustrations isn't perfect realism, but representation, and I think the slightly tilted top and flat base give enough of an impression to signal: "graduation cap."

I also tried the half shadow in the same color as the other shadow. I do like this better than the full shadow.

![5324c02e-d970-11e4-9453-54ed8ea3c038](https://cloud.githubusercontent.com/assets/10562544/6983798/1d4838ac-d9f0-11e4-849e-1f5ad16df3e0.png)

Here is just the half-shadow version:

![screen shot 2015-04-03 at 11 01 21 am](https://cloud.githubusercontent.com/assets/10562544/6983879/cdd726ba-d9f0-11e4-81c3-d6f64d2a7913.png)

**Comment by designlanguage** [comment]
@nataliafitzgerald, @ajbush, and I worked more on the graduation cap and here are the iterations leading to the version we feel best about.

This is where we left off previously:
![screen shot 2015-04-23 at 3 19 48 pm](https://cloud.githubusercontent.com/assets/10562544/7305893/747834d0-e9cf-11e4-8456-455066c21ff0.png)

Since grad caps are generally tilted up at the front (at least when worn), we wanted to see if this type of cap could still work.

![screen shot 2015-04-23 at 3 20 03 pm](https://cloud.githubusercontent.com/assets/10562544/7306522/d1cf0baa-e9d3-11e4-9155-f905b4276c1e.png)

If the cap sits on a flat surface, the long tassel could look strange dropping below it. We also thought two levels of shadow could allow us to define the cap shape a bit better.

![screen shot 2015-04-23 at 3 20 27 pm](https://cloud.githubusercontent.com/assets/10562544/7306535/eaa95ce8-e9d3-11e4-97fd-bc21fdc25360.png)

We also wanted to make sure the tassel read correctly, so we tried making it more complex. 

![screen shot 2015-04-23 at 3 20 21 pm](https://cloud.githubusercontent.com/assets/10562544/7306638/b517b8ee-e9d4-11e4-84a2-dc83b9fc35b2.png)

We then tried to flatten out the base of the simplified tassel and found that helped just as well. So this is where we've arrived at:

![screen shot 2015-04-23 at 4 24 48 pm](https://cloud.githubusercontent.com/assets/10562544/7306707/4c2afeb2-e9d5-11e4-9f67-7bedfd145535.png)

**Comment by nataliafitzgerald** [comment]
@designlanguage - Thanks for documenting your process. I'm actually liking both of the last two options. In the second to last option, you don't include the little piece of tassel string on top of the hat like you do in the last. 

I went ahead and tried this out in new identity page (since we're using the grad cap there). One thing I wonder is whether we would want to go any taller with the hat part of this so that it fills the space a little better. I feel like this could still look true to life if it was slightly taller. I'm not sure how this would work but just something to consider. 

![screen shot 2015-04-23 at 5 41 49 pm](https://cloud.githubusercontent.com/assets/6887128/7308181/391d4086-e9e0-11e4-9b71-5dcbc2a4bb3c.png)

![screen shot 2015-04-23 at 5 41 25 pm](https://cloud.githubusercontent.com/assets/6887128/7308192/4a2a92ca-e9e0-11e4-9cb7-419598adbbb1.png)

**Comment by designlanguage** [comment]
@nataliafitzgerald - Here are the two options revised. I made the hats a bit taller, as well as slightly increasing the angle of the tilt. That should help fill the space better. I also brought the grays back to our palette and increased the contrast of the grays a bit. Of the two, I prefer the one on the right. The tassel feels more defined and the detail in the one on the left can get a little lost when it's shown smaller, as you have it in the above post.

![screen shot 2015-04-24 at 9 26 14 am](https://cloud.githubusercontent.com/assets/10562544/7319630/d7f27332-ea64-11e4-85c1-f5d004331ff4.png)

**Comment by ajbush** [comment]
These look really nice @designlanguage!

I kinda prefer the tassel on the left but both are nice. Maybe adding that 45° shadow beneath the knot would help it pop.

I wonder how it would look if the thickness of the top/square part of the cap was about half the thickness/height it is now? Maybe the corners could be more pointed like somewhere between where you landed here and your very [first post](https://cloud.githubusercontent.com/assets/10562544/6983744/973eaf66-d9ef-11e4-976b-9ba61fd0dd85.png).

**Comment by designlanguage** [comment]
Thanks @ajbush!

Yeah, a shadow on the complex tassel definitely helps. Also here are three cap thicknesses. I agree that the thinner (middle) version feels better. I also wanted to compare a flat brim like the old grad cap just to test it. Not a fan of that one.

![screen shot 2015-04-24 at 10 19 23 am](https://cloud.githubusercontent.com/assets/10562544/7320653/f114fad6-ea6b-11e4-81d9-70171dbb9712.png)

**Comment by nataliafitzgerald** [comment]
@designlanguage - I like the second option. This one: 

![screen shot 2015-04-24 at 3 22 21 pm](https://cloud.githubusercontent.com/assets/6887128/7326753/bda9010e-ea95-11e4-84b6-473fa188f6ea.png)
Unless there are any further necessary adjustments, this is approved.

In order to add it to the illustration library, you will need to create an RGB file and a CMYK file. I made a folder for this here: https://drive.google.com/a/cfpb.gov/folderview?id=0B3epifaWXQGRfkVtLVFwSHU4SGdZVnhSVFM0Z3dsWG0wbUo3YjY5RWh5Nl92OHhpZVB4NDA&usp=sharing

**Comment by designlanguage** [comment]
Thanks @nataliafitzgerald. Both the CMYK and the RGB versions have been uploaded.

**Comment by nataliafitzgerald** [comment]
@designlanguage - These look great! 

Make sure you add the date to all files that are uploaded to our templates. The file naming conventions for templates can be found here (under Brand 2.0):  https://[GHE]/design-ops/team/wiki/Folder-and-file-naming-conventions. 

It looks as though our other illustration files are all missing the "cfpb_" All our other templates include this, though. So, we should probably follow this practice here as well (unless there is a compelling reason why we don't). I'd like @ReedOKelly to comment on this since he built out the illustration library when we transferred it to the new Drive. Reed?

**Comment by ReedOKelly** [comment]
@designlanguage - I agree, this is great work.
@nataliafitzgerald - With regards to the names, I only changed the dates. When I updated the files, they did not have the "cfpb_" prefix, and I may not have been aware at the time that it should be added. I should be able to remedy this soon. Let me know if there are any other changes that should be implemented while I'm at it.

**Comment by designlanguage** [comment]
Thanks @nataliafitzgerald and @ReedOKelly! The dates were already added to the filenames, but I added an underscore between GradCap and the color mode.

**Comment by nataliafitzgerald** [comment]
@ReedOKelly - As a general rule for illustration file naming (only), I am fine with omitting the cfpb_ since these illustrations are links/a part of a file rather than full files. Let's go ahead and make sure to use sentence case, however, for overall consistency with our naming conventions. 

@designlanguage - Yes, I added the dates earlier today. And I went ahead and changed this one to sentence case. So, we should be good to go!

@Dnpizarro - If we have updated the illustration library can we go ahead and close out this issue? I just want to be sure we follow the proper process. Thanks!
